diff -ruw openexr-2.2.1.old/IlmBase/Half/Makefile.am openexr-2.2.1.new/IlmBase/Half/Makefile.am
--- openexr-2.2.1.old/IlmBase/Half/Makefile.am	2017-11-17 23:31:37.000000000 +0000
+++ openexr-2.2.1.new/IlmBase/Half/Makefile.am	2019-08-22 20:07:18.384147738 +0100
@@ -15,18 +15,18 @@
 # these are used to build eLut.h and toFloat.h dynamically
 EXTRA_DIST = eLut.cpp toFloat.cpp CMakeLists.txt
 
-CLEANFILES = eLut eLut.h toFloat toFloat.h
+CLEANFILES = eLut$(EXEEXT) eLut.h toFloat$(EXEEXT) toFloat.h
 
 eLut_SOURCES = eLut.cpp
 
 toFloat_SOURCES = toFloat.cpp
 
-eLut.h:	eLut
-	./eLut > eLut.h
+eLut.h:	eLut$(EXEEXT)
+	./eLut$(EXEEXT) > eLut.h
 
-toFloat.h: toFloat
-	   ./toFloat > toFloat.h
+toFloat.h: toFloat$(EXEEXT)
+	   ./toFloat$(EXEEXT) > toFloat.h
 
 BUILT_SOURCES = eLut.h toFloat.h
 
-noinst_PROGRAMS = eLut toFloat
+noinst_PROGRAMS = eLut$(EXEEXT) toFloat$(EXEEXT)
diff -ruw openexr-2.2.1.old/OpenEXR/IlmImf/dwaLookups.cpp openexr-2.2.1.new/OpenEXR/IlmImf/dwaLookups.cpp
--- openexr-2.2.1.old/OpenEXR/IlmImf/dwaLookups.cpp	2017-11-17 23:31:37.000000000 +0000
+++ openexr-2.2.1.new/OpenEXR/IlmImf/dwaLookups.cpp	2019-08-22 20:11:24.608871524 +0100
@@ -436,7 +436,7 @@
 int
 cpuCount()
 {
-    if (!IlmThread::supportsThreads()) return 1;
+    return 1;
 
     int cpuCount = 1;
 
@@ -492,6 +492,7 @@
         }
     }
 
+#if 0
     if (IlmThread::supportsThreads()) {
         std::vector<LutHeaderWorker::Runner*> runners;
         for (size_t i=0; i<workers.size(); ++i) {
@@ -501,7 +502,9 @@
         for (size_t i=0; i<workers.size(); ++i) {
             delete runners[i];
         }
-    } else {
+    } else
+#endif
+    {
         for (size_t i=0; i<workers.size(); ++i) {
             workers[i]->run(i == 0);
         }
diff -ruw openexr-2.2.1.old/OpenEXR/IlmImf/ImfOptimizedPixelReading.h openexr-2.2.1.new/OpenEXR/IlmImf/ImfOptimizedPixelReading.h
--- openexr-2.2.1.old/OpenEXR/IlmImf/ImfOptimizedPixelReading.h	2017-11-17 23:31:37.000000000 +0000
+++ openexr-2.2.1.new/OpenEXR/IlmImf/ImfOptimizedPixelReading.h	2019-08-22 17:29:56.810442769 +0100
@@ -70,7 +70,7 @@
 bool
 isPointerSSEAligned (const void* EXR_RESTRICT pPointer)
 {
-    unsigned long trailingBits = ((unsigned long)pPointer) & 15;
+    unsigned long long trailingBits = ((unsigned long long)pPointer) & 15;
     return trailingBits == 0;
 }
 
diff -ruw openexr-2.2.1.old/OpenEXR/IlmImf/ImfSystemSpecific.h openexr-2.2.1.new/OpenEXR/IlmImf/ImfSystemSpecific.h
--- openexr-2.2.1.old/OpenEXR/IlmImf/ImfSystemSpecific.h	2017-11-17 23:31:37.000000000 +0000
+++ openexr-2.2.1.new/OpenEXR/IlmImf/ImfSystemSpecific.h	2019-08-22 20:09:32.784556017 +0100
@@ -62,9 +62,7 @@
 
 static void* EXRAllocAligned(size_t size, size_t alignment)
 {
-    void* ptr = 0;
-    posix_memalign(&ptr, alignment, size);
-    return ptr;
+    return malloc(size);
 }
 
 
diff -ruw openexr-2.2.1.old/OpenEXR/IlmImf/Makefile.am openexr-2.2.1.new/OpenEXR/IlmImf/Makefile.am
--- openexr-2.2.1.old/OpenEXR/IlmImf/Makefile.am	2017-11-17 23:31:37.000000000 +0000
+++ openexr-2.2.1.new/OpenEXR/IlmImf/Makefile.am	2019-08-22 17:16:25.623943824 +0100
@@ -192,20 +192,20 @@
 	   -I$(top_builddir)  \
 	   -I$(top_srcdir)/config
 
-CLEANFILES = b44ExpLogTable b44ExpLogTable.h dwaLookups dwaLookups.h
+CLEANFILES = b44ExpLogTable$(EXEEXT) b44ExpLogTable.h dwaLookups$(EXEEXT) dwaLookups.h
 
 b44ExpLogTable_SOURCES = b44ExpLogTable.cpp
 b44ExpLogTable_LDADD = @ILMBASE_LDFLAGS@ @ILMBASE_LIBS@
 
-b44ExpLogTable.h: b44ExpLogTable
-	./b44ExpLogTable > b44ExpLogTable.h
+b44ExpLogTable.h: b44ExpLogTable$(EXEEXT)
+	./b44ExpLogTable$(EXEEXT) > b44ExpLogTable.h
 
 dwaLookups_SOURCES = dwaLookups.cpp
 dwaLookups_LDADD = @ILMBASE_LDFLAGS@ @ILMBASE_LIBS@
 
-dwaLookups.h: dwaLookups
-	./dwaLookups > dwaLookups.h
+dwaLookups.h: dwaLookups$(EXEEXT)
+	./dwaLookups$(EXEEXT) > dwaLookups.h
 
 BUILT_SOURCES = b44ExpLogTable.h dwaLookups.h
 
-noinst_PROGRAMS = b44ExpLogTable dwaLookups
+noinst_PROGRAMS = b44ExpLogTable$(EXEEXT) dwaLookups$(EXEEXT)
diff -ruw openexr-2.2.1.old/OpenEXR/Makefile.am openexr-2.2.1.new/OpenEXR/Makefile.am
--- openexr-2.2.1.old/OpenEXR/Makefile.am	2017-11-17 23:31:37.000000000 +0000
+++ openexr-2.2.1.new/OpenEXR/Makefile.am	2019-08-22 20:02:42.755270220 +0100
@@ -4,9 +4,7 @@
 # (an alternative to the acinclude.m4 mechanism)
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = config IlmImf IlmImfUtil IlmImfTest IlmImfUtilTest \
-	  IlmImfFuzzTest exrheader exrmaketiled IlmImfExamples doc \
-	  exrstdattr exrmakepreview exrenvmap exrmultiview exrmultipart
+SUBDIRS = config IlmImf IlmImfUtil
 
 DIST_SUBDIRS = \
 	$(SUBDIRS) 
